packages/homepage/content/articles/codesadbox-introducing-improved-crash-reporting/index.md

```diff
@@ -1,32 +1,33 @@
 ---
 featuredImage: ./images/banner.png
-slug: CodeSadbox
```

Comment:
Will this break the existing url?

Reply:
It does indeed.
In #2011 [I did that too](https://github.com/codesandbox/codesandbox-client/pull/2011/files#diff-d725083d80d8ad87687c62592c2a2b72R3) with the `Templates` article.

Can revert these 2 back for now if you want, since there's another followup that can redirect these to the new slug

> Change `/post/blog-slug` to `/blog/blog-slug` an redirect all old links to the new ones

Reply:
Cool! Yeah, let's do the redirects in the future!